dist/tools/zep_dispatch/main.c

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Maybe unpopular proposal, but why not implement this in `python`? You wouldn't need to compile it and overall I think this would be way simpler.

Reply:
Mostly because I'm more comfortable with C than with Python ;-)